readme.markdown

```diff
@@ -217,3 +217,30 @@ We have received reports that the `idris-stay-in-current-window-on-compiler-erro
     (dolist (x '("*idris-notes*" "*idris-holes*" "*idris-info*"))
       (plist-put (cdr (assoc x popwin:special-display-config)) :noselect t))))
 ```
+
+
+### [`frames-only-mode`](https://github.com/davidshepherd7/frames-only-mode) and excessive buffer pop-ups
+
+[`frames-only-mode`](https://github.com/davidshepherd7/frames-only-mode) obviates the need for emacs internal windows so that emacs can get along better with tiling managers - such as [xmonad](https://github.com/xmonad/xmonad) - by using emacs' frames instead of windows.
+
+Throughout a session with `idris-mode`, many frames will accummulate, such as `*idris-holes*`, and over time these clutter your screen. A quick simple solution is to add the following to your emacs configuration:
+
+```elisp
+(add-hook 'idris-mode-hook
+          (lambda ()
```

Comment:
As a matter of elisp style, I think that this should be defined as a `defun`, and the symbol added to the hook. This makes it easier for users to remove it from the hook again later, and provides better semantics when the user wants to iteratively tweak this definition.